### PR TITLE
ODF template update

### PR DESCRIPTION
**Pull request by gayathrimenath-ibm**

### CODE REVIEW

File: config-templates/redhat/odf-local/changelog.json [hunk 1]
```diff
@@ -3,93 +3,91 @@
         "4.12": {
             "status": "supported",
             "latest-revision": {
+                "revision": "3",
+                "release-date": "17 May 2023",
```
Comment (mssachan) [inline config-templates/redhat/odf-local/changelog.json]: Change release date to `19 May 2023` at all places.

File: config-templates/redhat/odf-local/changelog.json [hunk 2]
```diff
@@ -3,93 +3,91 @@
...
+                "cves": "CVE-2023-27535, CVE-2022-36227, CVE-2022-3204, CVE-2022-35252, CVE-2022-43552, CVE-2023-29400, CVE-2023-24540,CVE-2023-24539",
+                "golang": "1.19.9",
+                "ubi": "8.8-860",
+                "new-features": [],
+                "fixes": [""]
```
Comment (mssachan) [inline config-templates/redhat/odf-local/changelog.json]: `"fixes": [""]` ---> `"fixes": []` at all places.

File: config-templates/redhat/odf-remote/changelog.json [hunk 3]
```diff
@@ -3,93 +3,91 @@
...
-                ],
-                "fixes": []
             }
         }
     },
     {
         "4.11": {
             "status": "supported",
             "latest-revision": {
+                "revision": "3",
```
Comment (mssachan) [inline config-templates/redhat/odf-remote/changelog.json]: Fix revision number here

File: config-templates/redhat/odf-remote/changelog.json [hunk 4]
```diff
@@ -3,93 +3,91 @@
...
-                "new-features": [],
-                "fixes": []
             }
         }
     },
     {
         "4.10": {
             "status": "supported",
             "latest-revision": {
+                "revision": "3",
```
Comment (mssachan) [inline config-templates/redhat/odf-remote/changelog.json]: Fix revision number here

File: config-templates/redhat/odf-remote/changelog.json [hunk 5]
```diff
@@ -3,93 +3,91 @@
...
-                "new-features": [],
-                "fixes": []
             }
         }
     },
     {
         "4.9": {
             "status": "supported",
             "latest-revision": {
+                "revision": "3",
```
Comment (mssachan) [inline config-templates/redhat/odf-remote/changelog.json]: Fix revision number here

File: config-templates/redhat/odf-local/changelog.json [hunk 6]
```diff
@@ -3,93 +3,91 @@
         "4.12": {
             "status": "supported",
             "latest-revision": {
+                "revision": "3",
+                "release-date": "17 May 2023",
+                "cves": "CVE-2023-27535, CVE-2022-36227, CVE-2022-3204, CVE-2022-35252, CVE-2022-43552, CVE-2023-29400, CVE-2023-24540,CVE-2023-24539",
+                "golang": "1.19.9",
```
Comment (mssachan) [inline config-templates/redhat/odf-local/changelog.json:9]: Just verify if you have fixed following PSIRTs as well with the new images?
https://github.ibm.com/alchemy-containers/armada-storage/issues/4484
https://github.ibm.com/alchemy-containers/armada-storage/issues/4508
https://github.ibm.com/alchemy-containers/armada-storage/issues/4487
If YES, please add corresponding CVEs as well in the list.

**Review by mssachan — CHANGES_REQUESTED** [review]

File: config-templates/redhat/odf-local/changelog.json (hunk 6 above)
Comment (gayathrimenath-ibm) [reply config-templates/redhat/odf-local/changelog.json:9]: Not added

**Comment by gayathrimenath-ibm** [comment]
@mssachan could you please verify

**Review by mssachan — APPROVED** [review]
LGTM

**Pull request merged**